nltk/classify/rte_classify.py

```diff
@@ -32,7 +32,7 @@ def ne(token):
         return True
     return False
 
-def lemmatize(word):
+def _lemmatize(word):
```

Comment:
(in any case this`_lemmatize()` change is merely a suggestion, I don't know about naming convention or indeed possibly existing documentation referring to `lemmatize`)